js/app.js

```diff
@@ -0,0 +1,13 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
+const bar2 = document.querySelector(".bar2")
+
+hamburger.addEventListener("click", () => {
+  hamburger.classList.toggle("active");
+  navMenu.classList.toggle("active");
+})
+
+document.querySelectorAll(".nav-link").forEach(n => n.addEventListener("click", () => {
```

Comment:
- Kindly run the command to install Eslint so that errors can be caught and fixed. 😉